src/components/WeekDayPicker/styled/index.js

```diff
@@ -0,0 +1,94 @@
+import styled from 'styled-components';
+import attachThemeAttrs from '../../../styles/helpers/attachThemeAttrs';
+import HelpText from '../../Input/styled/helpText';
+
+const StyledFieldset = styled.fieldset`
+    line-height: 1.5rem;
+    margin: 0;
+    padding: 0;
+    border: 0;
+    box-sizing: border-box;
+    text-align: center;
+`;
+
+const StyledHelpText = styled(HelpText)`
+    text-align: center;
+`;
+
+const StyledWeekDayContainer = styled.span`
+    line-height: inherit;
+    height: inherit;
+
+    :not(:last-child) {
+        margin-right: 0.25rem;
+    }
+`;
+
+const StyledWeekDayLabel = attachThemeAttrs(styled.label).attrs(props => {
+    const { palette } = props;
+
+    const borderColor = palette.border.divider;
+    const defaultColor = palette.text.label;
+    const defaultBackground = palette.background.main;
+    const activeColor = palette.getContrastText(palette.brand.main);
+    const activeBackground = palette.brand.main;
+
+    return {
+        borderColor,
+        defaultColor,
+        defaultBackground,
+        activeColor,
+        activeBackground,
+    };
+})`
+    width: 28px;
+    height: 28px;
+    border-radius: 50%;
+    display: inline-block;
+    text-align: center;
+    font-size: 10px;
+    color: ${props => props.defaultColor};
+    background: ${props => props.defaultBackground};
+    border: 1px solid ${props => props.borderColor};
+    transition: all 0.2s ease;
+
+    ${props =>
+        !props.readOnly &&
+        !props.disabled &&
```

Comment:
If this is neither `readOnly` nor `disabled` then it is default right,
why we need to do this check?

Reply:
Totally makes sense, even I was shocked for not seeing hahahaha (first thought).

It is needed for the case when user sends specific availableDates array.

So we need the hover action only on enabled days... this is because programmatically is being disabled the days sent by the user.

Another solution might be having the default hover block without restrictions and then I will need to have an extra control to disable the days passed by user. I thought that this solution is shorter and cleaner.